src/app/shared/nav-bar/nav-bar.styles.css

```diff
@@ -48,3 +48,12 @@
   text-decoration: none;
 
 }
+
+/* Breakpoint for sticky mobile-nav */
+@media only screen and (max-width: 768px) {
+  nav {
+    position: sticky !important;
```

Comment:
For some reason `position: sticky;` works differently on IOS. As such, the same error (the navbar floating away when the user scrolls) is still happening. Slack me if you have questions.

Reply:
I just found [this site](https://caniuse.com/#feat=css-sticky) while i was looking it up. it might be useful in the future too. it looks like it just needs a webkit prefix, I'll check it out in a moment.